tests/datasets/test_deepglobelandcover.py

```diff
@@ -0,0 +1,76 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import os
+import shutil
+from pathlib import Path
+
+import matplotlib.pyplot as plt
+import pytest
+import torch
+import torch.nn as nn
+from _pytest.fixtures import SubRequest
+from _pytest.monkeypatch import MonkeyPatch
+
+from torchgeo.datasets import DeepGlobeLandCover
+
+
+class TestDeepGlobeLandCover:
+    @pytest.fixture(params=["train", "test"])
+    def dataset(
+        self, monkeypatch: MonkeyPatch, request: SubRequest
+    ) -> DeepGlobeLandCover:
+        md5 = "2cbd68d36b1485f09f32d874dde7c5c5"
+        splits = ["train", "test"]
+        monkeypatch.setattr(DeepGlobeLandCover, "md5", md5)
+        monkeypatch.setattr(DeepGlobeLandCover, "splits", splits)
+        root = os.path.join("tests", "data", "deepglobelandcover")
+        split = request.param
+        transforms = nn.Identity()  # type: ignore[no-untyped-call]
```

Comment:
```suggestion
        transforms = nn.Identity() 
```
Latest version of PyTorch adds type hints for this